src/behavioural-events/sdk-event-helpers.ts

```diff
@@ -33,15 +31,15 @@ export function createCheckoutFlowErrorEvent({
   };
 }
 
-export function createCheckoutBillingFormErrorEvent({
+export function createCheckoutPaymentFormErrorEvent({
   errorCode,
   errorMessage,
 }: {
   errorCode: string | null;
   errorMessage: string;
-}): CheckoutBillingFormErrorEvent {
+}): CheckoutPaymentFormErrorEvent {
   return {
-    eventName: SDKEventName.CheckoutBillingFormError,
+    eventName: SDKEventName.CheckoutPaymentFormError,
```

Comment:
Are we deprecating the `CheckoutBillingFormError` event type in favor of `CheckoutPaymentFormError`?

Reply:
The name convention indicates the page. Given that there is no billing form anymore I prefer to use `CheckoutPaymentFormError`.